docs/06-concepts/11-authentication/02-basics.md

```diff
@@ -96,3 +96,37 @@ class MyEndpoint extends Endpoint {
 :::caution
 Keep in mind that a scope is merely an arbitrary string and can be written in any format you prefer. However, it's crucial to use unique strings for each scope, as duplicated scope strings may lead to unintentional data exposure.
 :::
+
+## User authentication
+
+The `StatusEndpoint` class includes methods for handling user sign-outs, whether from a single device or all devices.
+
+- **Sign out device**
+
+    Use the `signOutDevice` method to log the user out of the current device only, closing streaming connections for this device while keeping other devices active.
```

Comment:
Suggestion: This signs out a single authentication token. And a single authentication token is often connect to a single device.